modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/shibari_stand.dm

```diff
@@ -0,0 +1,269 @@
+	shibari_rope_overlay_behind = mutable_appearance('modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_structures/shibari_stand.dmi', "ropes_behind_[color]", BELOW_MOB_LAYER)
+	add_overlay(shibari_rope_overlay)
+	add_overlay(shibari_rope_overlay_behind)
+
+/obj/structure/chair/shibari_stand/post_buckle_mob(mob/living/buckledmeme)
+	buckledmeme.pixel_y = buckledmeme.base_pixel_y + 4
+	buckledmeme.pixel_x = buckledmeme.base_pixel_x
+	buckledmeme.layer = BELOW_MOB_LAYER
+
+	if(LAZYLEN(buckled_mobs))
+		if(ishuman(buckled_mobs[1]))
+			current_mob = buckled_mobs[1]
+
+	if(current_mob)
+		if(current_mob.handcuffed)
+			current_mob.handcuffed.forceMove(loc)
+			current_mob.handcuffed.dropped(current_mob)
+			current_mob.set_handcuffed(null)
+			current_mob.update_handcuffed()
+		current_mob.set_handcuffed(new /obj/item/restraints/handcuffs/milker/shibari(current_mob))
+		current_mob.handcuffed.parented_struct = src
+		if(HAS_TRAIT(current_mob, TRAIT_ROPEBUNNY))
+			current_mob.handcuffed.breakouttime = 4 MINUTES
+		current_mob.update_abstract_handcuffed()
+
+//Restore the position of the mob after unbuckling.
+/obj/structure/chair/shibari_stand/post_unbuckle_mob(mob/living/buckledmeme)
+	buckledmeme.pixel_x = buckledmeme.base_pixel_x + buckledmeme.body_position_pixel_x_offset
+	buckledmeme.pixel_y = buckledmeme.base_pixel_y + buckledmeme.body_position_pixel_y_offset - 4
+	buckledmeme.layer = initial(buckledmeme.layer)
+
+	cut_overlay(shibari_shadow_overlay)
+	cut_overlay(shibari_rope_overlay)
+	cut_overlay(shibari_rope_overlay_behind)
+
+	if(current_mob)
+		if(current_mob.handcuffed)
+			current_mob.handcuffed.dropped(current_mob)
+		current_mob.set_handcuffed(null)
+		current_mob.update_abstract_handcuffed()
+
+	if(ropee)
+		ropee.forceMove(get_turf(src))
+	current_mob = null
+
+/obj/item/restraints/handcuffs/milker/shibari
+	name = "ropes"
+	desc = "A shibari rope for restraining hands."
+	breakouttime = 2 MINUTES
+
+///////////////////////////
+//s-stand construction kit/
+///////////////////////////
+
+/obj/item/shibari_stand_kit
+	name = "shibari stand construction kit"
+	desc = "Construction requires a wrench."
+	icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_structures/bdsm_furniture.dmi'
+	throwforce = 0
+	icon_state = "shibari_kit"
+	w_class = WEIGHT_CLASS_HUGE
+	var/current_color = "pink"
+	var/color_changed = FALSE
+	var/static/list/shibarikit_designs
+
+/obj/item/shibari_stand_kit/proc/populate_shibarikit_designs()
+	shibarikit_designs = list(
+		"pink" = image (icon = src.icon, icon_state = "shibari_kit_pink"),
+		"teal" = image (icon = src.icon, icon_state = "shibari_kit_teal"),
+		"black" = image (icon = src.icon, icon_state = "shibari_kit_black"),
+		"red" = image (icon = src.icon, icon_state = "shibari_kit_red"),
+		"white" = image (icon = src.icon, icon_state = "shibari_kit_white"))
```

Comment:
If you're going to go with that, I highly suggest once again to use GAGS.

Reply:
I still want this to be turned into GAGS, the amount of repetition is just insane, that would all be solved with GAGS.